Common/ExecutionModel/vtkThreadedImageAlgorithm.cxx

```diff
@@ -274,6 +369,12 @@ int vtkThreadedImageAlgorithm::RequestData(
           vtkInformation* info = portInfo->GetInformationObject(j);
           str.Inputs[i][j] = vtkImageData::SafeDownCast(
             info->Get(vtkDataObject::DATA_OBJECT()));
+          
+            if(!updateExtentsFound && this->UseSmp)
```

Comment:
I am not too sure if grabbing the extents like this is exactly correct if there is no output extent.  I followed what the vtkThreadedImageAlgorithmThreadedExecute, grabbing an extent from the input.